doozer/doozerlib/cli/inspect_stream.py

```
@@ -77,25 +87,32 @@ async def inspect_stream(runtime: Runtime, code: AssemblyIssueCode, strict: bool
 
 
 def _check_inconsistent_rhcos_rpms(
-    runtime: Runtime, assembly_inspector: AssemblyInspector, payload_generator: PayloadGenerator
+    runtime: Runtime,
+    assembly_inspector: AssemblyInspector,
+    payload_generator: PayloadGenerator,
+    registry_config: str = None,
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)